android/app/src/main/java/org/fossasia/openevent/api/Urls.java

```diff
@@ -46,6 +50,12 @@ public static String getBaseUrl() {
     }
 
     public static void setBaseUrl(String baseUrl) {
-        BASE_URL = baseUrl;
+        if (URLUtil.isValidUrl(baseUrl)) {
```

Comment:
So basically anything that is a valid URL will be qualified as good to go here tight?
So what if I use www.google.com here?
It is a valid URL but is not a valid API.
@mananwason